hrjobcontract/js/test/services/utils_test.js

```diff
@@ -0,0 +1,51 @@
+define([
+    'common/angular',
+    'common/angularMocks',
+    'job-contract/app'
+], function (angular, Mock) {
+    'use strict';
```

Comment:
i don't remember doing that :| if i did it i apologies, but the agreement [see here](https://compucorp.atlassian.net/browse/PCHR-1022) was to have 2 spaces for everything. does the .editorconfig discriminates based on the file type though? You're saying it works for php files but not for js files?